canvas_modules/harness/cypress/e2e/canvas/zoom.cy.js

```diff
@@ -138,22 +141,22 @@ describe("Test zoomToReveal function returns the appropriate zoom object for a l
 		cy.setXPercentOffset(55);
 		cy.setYPercentOffset(55);
 		cy.submitAPI();
+		cy.wait(500);
 		cy.verifyCanvasTransform("translate(-51.59999999999991,87.95000381469731) scale(1)");
 	});
 
 	it("Test ZoomTo function with x,y parameters after zooming out", function() {
 		cy.verifyCanvasTransform(undefined);
 		// Zooming the canvas.
 		cy.clickToolbarZoomOut();
-		// TODO -- write function to test these +/- 2 px.
-		// cy.verifyCanvasTransform("translate(45.81818181818181,32.22727272727275) scale(0.9090909090909091)");
+		cy.verifyCanvasTransform("translate(45.81818181818181,32.22727272727275) scale(0.9090909090909091)");
 		// Test get Zoom to reveal & ZoomTo
 		cy.selectEntryFromDropdown("Binding (entry) node-Execution node");
 		cy.setXPercentOffset(70);
 		cy.setYPercentOffset(60);
 		cy.submitAPI();
-		// TODO -- write function to test these +/- 2 px.
-		// cy.verifyCanvasTransform("translate(498.32727272727266,290.85454545454536) scale(0.9090909090909091)");
+		cy.wait(500);
+		cy.verifyCanvasTransform("translate(498.32727272727266,290.85454545454536) scale(0.9090909090909091)");
```

Comment:
Did you test to see if your code will return an error if an incorrect value is provided? I tried changing this line to:
```
cy.verifyCanvasTransform("translate(200,290.85454545454536) scale(0.9090909090909091)");
``` 
That is, with 200 as the x parameter and the test passed! 

When writing any tests, it is just as important to make sure the test will fail, in an invalid situation,  as it is to make sure the test passes.  If the test doesn't fail in the invalid situation then there would be no reason to run the test in the first place -- right?

Reply:
I can see this error is occurring due to logic of checking the variable type, whether its a string or not, changed with 
if (typeof movString === "string") ,
now if the pixel range is other than +/-2 px, the test will fail
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/d3d7012a-332c-4830-a7fd-6529f94298d2">